pgmpy/tests/test_factors/test_continuous/test_ContinuousFactor.py

```diff
@@ -31,6 +32,14 @@ def test_class_init(self):
         self.assertEqual(phi3.scope(), ['x', 'y', 'z'])
         self.assertEqual(phi3.pdf, self.pdf3)
 
+    def test_class_init_gaussian(self):
+        mean = np.array([1, -3, 4])
+        cov = np.array([[4, 2, -2], [2, 5, -5], [-2, -5, 8]])
+        phi1 = ContinuousFactor(['x1', 'x2'], 'gaussian', mean=mean, covariance=cov)
```

Comment:
@JustinTervala This should be a 3 variable distribution. Please add one more variable. Tests should pass then.